tomb/src/types/config/bucket/sync.rs

```diff
@@ -73,21 +74,18 @@ pub async fn determine_sync_state(
         let local_root_cid = local.metadata.get_root().map(|cid| cid.to_string());
         // If the metadata root CIDs match
         if local_root_cid == Some(current_remote.root_cid) {
-            omni.sync_state = Some(SyncState::MetadataSynced);
-
             // If there is actually data in the local origin
-            if compute_directory_size(&local.origin)? > 100 {
+            let tolerance = 100;
```

Comment:
Could you explain what this tolerance is doing? If we have 56 bytes here for some reason and we discard it wouldn't that corrupt our state? Could this 100 byte threshold be why we're missing restorable data?